testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -1218,3 +1231,112 @@ def test_record_type_sel():
 
     assert len(u.select_atoms('name CA and not record_type HETATM')) == 30
     assert len(u.select_atoms('name CA and record_type HETATM')) == 2
+
+
+def test_element_sel():
+    # test auto-topattr addition of string
+    u = mda.Universe(PDB_elements)
+    assert len(u.select_atoms("element H")) == 8
+    assert len(u.select_atoms("same element as index 12")) == 8
+
+
+def test_chain_sel():
+    u = mda.Universe(PDB_elements)
+    assert len(u.select_atoms("chainID A")) == len(u.atoms)
+
+
+@pytest.fixture()
+def u_fake_masses():
+    u = mda.Universe(TPR)
+    u.atoms[-10:].masses = - (np.arange(10) + 1.001)
+    u.atoms[:5].masses = 0.1 * 3  # 0.30000000000000004
+    u.atoms[5:10].masses = 0.30000000000000001
+    return u
+
+
+@pytest.mark.parametrize("selstr,n_atoms, selkwargs", [
+    ("mass 0.8 to 1.2", 23844, {}),
+    ("mass 8e-1 to 1200e-3", 23844, {}),
+    ("mass -5--3", 2, {}),  # select -5 to -3
+    ("mass -5 --3", 2, {}),  # spacing
+    ("mass -5- -3", 2, {}),  # spacing
+    ("mass -3 : -5", 0, {}),  # wrong way around
```

Comment:
So this selects nothing rather than raise an error?  I'm trying to think what this should do, maybe it should automatically flip the values to select `-3 : -5`

Reply:
I think @orbeckst and I discussed this, and we decided to follow the behaviour of VMD etc which just select nothing when the ranges are the wrong way around.